components/ProjectPreview/index.js

```diff
@@ -1,18 +1,22 @@
 import Image from "next/image";
 import Link from "next/link";
 import styled from "styled-components";
+import { ImageWrapper } from "../../styles";
 
 export default function ProjectPreview({ project }) {
   return (
     <ProjectItem key={project._id}>
-      <h3>{project.name}</h3>
+      <ProjectPreviewTitle>{project.name}</ProjectPreviewTitle>
       <Link href={`/${project._id}`}>
-        <Image
-          src={project.image}
-          alt="project-image"
-          width={375}
-          height={375}
-        />
+        <ImageWrapper>
+          <Image
+            src={project.image}
+            alt="project-image"
+            width={350}
+            height="0"
+            style={{ width: "100", height: "auto" }}
```

Comment:
I am a bit confused regarding the `width 100` because tbh I have no clue how inline styling handles it without a meassurement unit like `px`, `%`, `pt`, etc. Anyways I just wanted to point that if you are uncertain about this matter as well maybe take a look at it again and see if it needs a measurement unit or can stay like this 😮 😄

Reply:
i want to keep the image its own aspect ratio. tying a lot of ways only that works.